### ISSUE TITLE
Dots reach the end but nothing happens.

### ISSUE THREAD

**Issue by ehamner** [BirdBrainTechnologies/flutter-kindle-sttr2#83]
(Observed in K and 1st pilots) Kids notice that the dots on the bottom of the screen progress as they do problems, but at the end, nothing happens and they just keep getting more problems.

What should happen?

One girl in 1st wanted to go to a harder level. However teachers have been monitoring the levels.

{Discussion Lauren, Emily, Sam} Maybe it can ask to lengthen their session or go on to a new session?

**Comment by tasota** [comment]
So, I recall from the designs that there were supposed to be dots on the exercises landing page. After the set of problems is finished, should it have some kind of "good job" prompt and then direct them back to that landing screen?

**Comment by bambibrewer** [comment]
I don't think we want to redirect them, since so far, the teachers have been telling them what section they should be on. Are the dots actually useful for us? Should we consider just removing them?

**Comment by tasota** [comment]
If the goal is to just keep working on problems, it probably makes sense to not have the dots in the exercise. But maybe it would be helpful/nice to have a counter so they know how many problems they have finished?

It could be fun to also have something that changes when they reach different thresholds (different messages or colors after finishing 10 problems in the exercise, 25 problems, etc)

**Comment by ehamner** [comment]
The teachers are interested in knowing how many problems they have done. They have been asking them to write the problems on their white board and not erase them (the kids often forget and erase them anyway). It would not help the teacher know what problems they have done, but a count does at least seem it it's going in the right direction.

**Comment by mo11yj** [comment]
Originally I imagined that when they reached the end of the set of ten they would be redirected back to the landing page; however, knowing that teachers want them to keep going in one section, I agree that doesn't make the most sense. 

I will take a stab at designing in a counter or some indication of how many questions they have done that can just keep incrementing up rather than re-setting.

**Comment by mo11yj** [comment]
Landing screen w/ counter: zpl.io/bPvQeZl

Interior screen w/ counter: zpl.io/V1zDdDk

On both screens, I don't think it should show a zero, so if the student hasn't completed any questions yet, just removing the circle and number all together seems best to me.

**Comment by ehamner** [comment]
I like those designs.

**Comment by bambibrewer** [comment]
Agreed, those look nice!

**Comment by lczito** [comment]
I like it, too!
